docs/how-to-guides/manage-instances/launch-customized-virtual-machines-with-multipass-and-cloud-init.md

```diff
@@ -0,0 +1,102 @@
+(how-to-guides-manage-instances-launch-customized-virtual-machines-with-multipass-and-cloud-init)=
+# Launch customized virtual machines with Multipass and cloud-init
+
+If you want to set up a virtual machine with a specific environment or configuration, you can use the launch command along with a custom cloud-init YAML file and an optional post-launch health check to ensure everything is working correctly.
```

Comment:
Make this a declarative sentence instead of a conditional. i.e. You can set up instances with a customized environment or configuration using the `launch` command along with a custom...